content/academy/apify_platform/getting_started/apify_client.md

```diff
@@ -0,0 +1,200 @@
+const dataset = client.dataset(run['defaultDatasetId'])
+</marked-tab>
+<marked-tab header="Python" lang="python">
+dataset = client.dataset(run['defaultDatasetId'])
+</marked-tab>
+```
+
+Finally, we can collect the items in the dataset by using the `dataset.downloadItems()` function in Node.js, and the `dataset.list_items()` function in Python, then log them to the console.
+
+```marked-tabs
+<marked-tab header="Node.js" lang="javascript">
+const items = await dataset.downloadItems('json');
+
+console.log(JSON.parse(Buffer.from(items).toString('utf-8')));
+</marked-tab>
+<marked-tab header="Python" lang="python">
+items = dataset.list_items().items
+
+print(items)
+</marked-tab>
+```
+
+> Notice that in the JavaScript example, we had to convert the `items` to a [`Buffer`](https://nodejs.org/api/buffer.html), then convert the Buffer to a string and parse it. This is because `dataset.downloadItems()` returns a buffer.
+
+The final code for running the actor and collecting its dataset items looks like this:
+
+```marked-tabs
+<marked-tab header="Node.js" lang="javascript">
+// client.js
+import { ApifyClient } from 'apify-client';
+
+const client = new ApifyClient({
+    token: 'YOUR_TOKEN',
+});
+
+const run = await client.actor('YOUR_USERNAME/adding-actor').call({
+    num1: 4,
+    num2: 2,
+});
+
+const dataset = client.dataset(run['defaultDatasetId'])
+
+const items = dataset.downloadItems('json');
+
+console.log(JSON.parse(Buffer.from(items).toString('utf-8')));
+</marked-tab>
+<marked-tab header="Python" lang="python">
+# client.py
+from apify_client import ApifyClient
+
+client = ApifyClient(token='YOUR_TOKEN')
+
+actor = client.actor('YOUR_USERNAME/adding-actor').call(run_input={
+    'num1': 4,
+    'num2': 2
+})
+
+dataset = client.dataset(run['defaultDatasetId'])
+
+items = dataset.list_items().items
+
+print(items)
+</marked-tab>
+```
+
+## [](#updating-actor) Updating an actor
+
+If you check the **Settings** tab within your **adding-actor**, you'll notice that the default memory being allocated to the actor is **2048 MB**. This is a bit overkill considering the fact that the actor is only adding two numbers together - **256 MB** would be much more reasonable. Also, we can safely say that the run should never take more than 20 seconds (even this is a generous number), and that the default of 3600 seconds is also overkill.
+
+Let's change these two actor settings via the Apify client using the `actor.update()` function! First, we'll create a pointer to our actor, similar to as before (except this time, we won't be using `.call()` at the end):
+
+```marked-tabs
+<marked-tab header="Node.js" lang="javascript">
+const actor = client.actor('YOUR_USERNAME/adding-actor')
+</marked-tab>
+<marked-tab header="Python" lang="python">
+actor = client.actor('YOUR_USERNAME/adding-actor')
+</marked-tab>
+```
+
+Then, we'll just call the `.update()` method on the `actor` variable we created and pass in our new **default run options**:
```

Comment:
I would add a link to docs of the update function and explain where and how to find the body, as it's probably not obvious. The last link to API docs was at the beginning of previous lesson I believe.


```
{
     defaultRunOptions: {
         build: 'latest',
         memoryMbytes: 256,
         timeoutSecs: 20,
     },
 }
 ```